wurst/systems/core/Classes.wurst

```diff
@@ -1,103 +1,103 @@
-..put(UNIT_PRIEST, ClassType.BASE_CLASS)
-..put(UNIT_BOOSTER, ClassType.SUB_CLASS)
-..put(UNIT_BOOSTER_1, ClassType.SUB_CLASS)
-..put(UNIT_MASTER_HEALER, ClassType.SUB_CLASS)
-..put(UNIT_MASTER_HEALER_1, ClassType.SUB_CLASS)
-..put(UNIT_SAGE, ClassType.SUPER_CLASS)
-..put(UNIT_THIEF, ClassType.BASE_CLASS)
-..put(UNIT_ESCAPE_ARTIST, ClassType.SUB_CLASS)
-..put(UNIT_CONTORTIONIST, ClassType.SUB_CLASS)
-..put(UNIT_ASSASSIN, ClassType.SUPER_CLASS)
-..put(UNIT_SCOUT, ClassType.BASE_CLASS)
-..put(UNIT_OBSERVER, ClassType.SUB_CLASS)
-..put(UNIT_RADAR_SCOUT, ClassType.SUB_CLASS)
-..put(UNIT_SPY, ClassType.SUPER_CLASS)
-..put(UNIT_GATHERER, ClassType.BASE_CLASS)
-..put(UNIT_RADAR_GATHERER, ClassType.SUB_CLASS)
-..put(UNIT_HERB_MASTER, ClassType.SUB_CLASS)
-..put(UNIT_OMNIGATHERER, ClassType.SUPER_CLASS)
-..put(UNIT_DRUNKEN_TROLL, ClassType.BASE_CLASS)
-..put(UNIT_TROLL_BRAWLER, ClassType.SUB_CLASS)
-..put(UNIT_TROLL_BREWMASTER, ClassType.SUB_CLASS)
-
-let trollBaseClass = new HashMap<int, int>()
-..put(UNIT_HUNTER, UNIT_HUNTER)
-..put(UNIT_HUNTER_1, UNIT_HUNTER)
-..put(UNIT_TRACKER, UNIT_HUNTER)
-..put(UNIT_WARRIOR, UNIT_HUNTER)
-..put(UNIT_JUGGERNAUT, UNIT_HUNTER)
-..put(UNIT_BEAST_MASTER, UNIT_BEAST_MASTER)
-..put(UNIT_BEAST_MASTER_1, UNIT_BEAST_MASTER)
-..put(UNIT_SHAPESHIFTER_WOLF, UNIT_BEAST_MASTER)
-..put(UNIT_SHAPESHIFTER_BEAR, UNIT_BEAST_MASTER)
-..put(UNIT_SHAPESHIFTER_PANTHER, UNIT_BEAST_MASTER)
-..put(UNIT_SHAPESHIFTER_TIGER, UNIT_BEAST_MASTER)
-..put(UNIT_CHICKEN_FORM, UNIT_BEAST_MASTER)
-..put(UNIT_ULTIMATE_FORM, UNIT_BEAST_MASTER)
-..put(UNIT_MAGE, UNIT_MAGE)
-..put(UNIT_MAGE_1, UNIT_MAGE)
-..put(UNIT_ELEMENTALIST, UNIT_MAGE)
-..put(UNIT_ELEMENTALIST_1, UNIT_MAGE)
-..put(UNIT_HYPNOTIST, UNIT_MAGE)
-..put(UNIT_DEMENTIA_MASTER, UNIT_MAGE)
-..put(UNIT_PRIEST, UNIT_PRIEST)
-..put(UNIT_BOOSTER, UNIT_PRIEST)
-..put(UNIT_BOOSTER_1, UNIT_PRIEST)
-..put(UNIT_MASTER_HEALER, UNIT_PRIEST)
-..put(UNIT_MASTER_HEALER_1, UNIT_PRIEST)
-..put(UNIT_SAGE, UNIT_PRIEST)
-..put(UNIT_THIEF, UNIT_THIEF)
-..put(UNIT_ESCAPE_ARTIST, UNIT_THIEF)
-..put(UNIT_CONTORTIONIST, UNIT_THIEF)
-..put(UNIT_ASSASSIN, UNIT_THIEF)
-..put(UNIT_SCOUT, UNIT_SCOUT)
-..put(UNIT_OBSERVER, UNIT_SCOUT)
-..put(UNIT_RADAR_SCOUT, UNIT_SCOUT)
-..put(UNIT_SPY, UNIT_SCOUT)
-..put(UNIT_GATHERER, UNIT_GATHERER)
-..put(UNIT_RADAR_GATHERER, UNIT_GATHERER)
-..put(UNIT_HERB_MASTER, UNIT_GATHERER)
-..put(UNIT_OMNIGATHERER, UNIT_GATHERER)
-..put(UNIT_DRUNKEN_TROLL, UNIT_DRUNKEN_TROLL)
-..put(UNIT_TROLL_BRAWLER, UNIT_DRUNKEN_TROLL)
-..put(UNIT_TROLL_BREWMASTER, UNIT_DRUNKEN_TROLL)
-
-public function getTrollClassType(unit hero) returns ClassType
-    return trollClassType.get(hero.getTypeId())
-
-public function getTrollClassType(int heroId) returns ClassType
-    return trollClassType.get(heroId)
-
-public function getTrollBaseClass(unit hero) returns int
-    return trollBaseClass.get(hero.getTypeId())
-
-public function getTrollBaseClass(int heroId) returns int
-    return trollBaseClass.get(heroId)

```

Comment:
Looks like we have a conflict here -- based on @jonathanmclaus's suggestion we're going with `getTroll*`